### PR TITLE
Add Color3.new empty constructor and Color3.fromRGB

### PR DESCRIPTION
**Pull request by Kampfkarren**

### CODE REVIEW

**Comment by LPGhatguy** [comment]
Sorry I let this sit for so long! I was on vacation when it was submitted and then it fell off of my notifications.

File: lib/types/Color3_spec.lua [hunk 1]
```diff
@@ -1,9 +1,29 @@
...
+		color.b
+	}
+end
+
 describe("types.Color3", function()
-	it("should have a constructor", function()
+	it("should have an empty constructor", function()
+		local color = Color3.new()
+
+		assert.not_nil(color)
```
Comment (LPGhatguy) [inline lib/types/Color3_spec.lua]: We should probably make sure that the result is `{0, 0, 0}` for the empty constructor

File: lib/types/Color3.lua [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
+function Color3.new(...)
+	if select("#", ...) == 0 then
+		return Color3.new(0, 0, 0)
+	else
+		local r, g, b = ...
+
+		return {
+			r = r,
+			g = g,
+			b = b
```
Comment (LPGhatguy) [inline lib/types/Color3.lua]: Can you add a trailing comma here like the original block? It isn't that important here, but it makes diffs cleaner and makes it easier to change line ordering.

**Comment by Kampfkarren** [comment]
I can't do your changes right now, don't have access to a PC. I'd be happy to do it when I get the chance.

**Comment by Kampfkarren** [comment]
@LPGhatguy Finished.

**Comment by Kampfkarren** [comment]
I...have no idea why that check isn't successful?

**Comment by LPGhatguy** [comment]
Weird, I don't understand why it would be failing either.

**Pull request merged**